draft-ietf-quic-ack-frequency.md

```diff
@@ -422,13 +422,10 @@ packet before an idle period to avoid waiting for the ack delay.
 
 ## Burst Mitigation
 
-Receiving an acknowledgement can allow a sender to release new packets into the
-network. If a sender is designed to rely on the timing of peer acknowledgments
-("ACK clock"), delaying acknowledgments can cause undesirable bursts of data
-into the network. A sender MUST limit such bursts. In keeping with Section 7.7
-of {{QUIC-RECOVERY}}, a sender can either employ pacing or cause a receiver to
-send an acknowledgement for at least each initial congestion window of received
-data.
+Acknowledging more packets in a single ACK frame can cause larger bursts
+of packets into the network. Senders MUST continue to follow Section 7.7
```

Comment:
I still don't think this needs normative language at all. We also don't say. You MUST implement RFC9000 before you implement this spec.

Reply:
I removed the MUST.